contracts/loan_pool/src/contract.rs

```diff
@@ -109,7 +112,7 @@ impl LoanPoolTrait for LoanPoolContract {
         */
         let address = String::from_str(
             &e,
-            "CCR7ARWZN4WODMEWVTRCMPPJJQKE2MBKUPJBSYWCDEOT3OLBPAPEGLPH",
+            "CB6MHNR6FJMQHJZDWOKAU4KESR4OARLPZ4RMN57R55P2QUBH4QJENHLY",
```

Comment:
Tää oli kaiketi väliaikanen ratkasu?

Reply:
Joo tämä juuri pitäisi korjata 'Store loan manager’s ID for liquidity pool' tiketissä.